job/common/src/main/java/alluxio/job/wire/JobWorkerHealth.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.job.wire;
+
+import alluxio.util.CommonUtils;
+
+import com.google.common.base.Objects;
+import com.google.common.collect.Lists;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * The job worker health information.
+ */
+public class JobWorkerHealth {
+
+  private final long mWorkerId;
+  private final ArrayList<Double> mLoadAverage;
+  private final long mLastUpdated;
+
+  private String mHostname;
+
+  /**
+   * Default constructor.
+   * @param workerId the worker id
+   * @param loadAverage the cpu load average
```

Comment:
nit: Can you be more descriptive about this argument or refer to where else in the javadoc I can read about this param?

Reply:
can we still update this?

Reply:
I don't expect this constructor to be called by anyone other than as output of CentralProcessor.getSystemLoadAverage so I guess I'll put that instead.